README.md

```diff
@@ -35,7 +35,8 @@ The following specifications are available:
 | **AMQP Event Format** | - | [master](https://github.com/cloudevents/spec/blob/master/amqp-format.md) |
 | **AMQP Transport Binding** | - | [master](https://github.com/cloudevents/spec/blob/master/amqp-transport-binding.md) |
 
-There is also the [CloudEvents Extension Attributes](https://github.com/cloudevents/spec/blob/master/extensions.md)
+There is also the
+[CloudEvents documented extension attributes](https://github.com/cloudevents/spec/blob/master/documented-extensions.md)
```

Comment:
This link "https://github.com/cloudevents/spec/blob/master/documented-extensions.md" is not a valid link. 
I see that some documented extension is put in the extensions.md. What is this document for? I would suggest that this link be added after the document is created and the WG has reviewed it.  We need to be consistent to avoid confusion.

Reply:
yes, it will be fixed once the PR is merged. For now its just within the scope of the branch/PR.